backend/gamecreation/views.py

```
@@ -6,35 +6,243 @@
-        board=PokerBoard.objects.create(game_name=request.data['gameName'],game_description=request.data['gameDescription'])
-        # for i in range(0,len(request.data.users)):
-            
-        pokermember.objects.create(poker=board, member=user)
-        return Response({'Game Created Successfully'},status.HTTP_201_CREATED)
+        email='abhijeet.kumar.josh.7985@gmail.com'
+        token=request.data.get('apiToken')
+        site=request.data.get('site')
+        user=request.user
+        importType=request.data.get('importType')
+        importValue=request.data.get('importValue')
+        api=ApiKeys.objects.create(user=user,apikey=request.data['apiToken'],cloudsite=request.data['site'])
```

Ruff (0.12.2)

Local variable `api` is assigned to but never used

Remove assignment to unused variable `api`

(F841)